File: Assets/Scripts/ControllBall.js
#pragma strict
var rotationSpeed = 100;
var moveSpeed = 1f;
var target : Transform;

function Update () {
	//Handle ball rotation
	var rotation : float = Input.GetAxis ("Horizontal") * rotationSpeed;
	rotation *= Time.deltaTime;
	GetComponent.<Rigidbody>().AddRelativeTorque (Vector3.back * rotation);
}

File: Assets/Scripts/ControllBall.cs
using UnityEngine;
using System.Collections;

[System.Serializable]
public partial class ControllBall : MonoBehaviour
{
    public int rotationSpeed;
    public float moveSpeed;
    public Transform target;
    public virtual void Update()
    {
        //Handle ball rotation
        float rotation = Input.GetAxis("Horizontal") * this.rotationSpeed;
        rotation = rotation * Time.deltaTime;
        this.GetComponent<Rigidbody>().AddRelativeTorque(Vector3.back * rotation);
    }

    public ControllBall()
    {
        this.rotationSpeed = 100;
        this.moveSpeed = 1f;
    }

}